po/de.po

```diff
@@ -1253,10 +1251,10 @@ msgstr "Sonstige Zahlungen"
 #. Translators: %s is "Taxes",
 #. "Insurance", or similar.
 #: gnucash/gnome/assistant-loan.cpp:766
-#, fuzzy, c-format
+#, c-format
 #| msgid "... pay \"%s\"?"
```

Comment:
Here and later. Together with the fuzzy flag drop the old msgid.
[Edit:]
In your Upload to weblate they are automatically removed. So we will finally use that version.